src/refda/adm/ietf_dtnma_agent.c

```diff
@@ -353,13 +365,45 @@ static void refda_adm_ietf_dtnma_agent_edd_num_msg_tx(refda_edd_prod_ctx_t *ctx)
      * |START CUSTOM FUNCTION refda_adm_ietf_dtnma_agent_edd_num_msg_tx BODY
      * +-------------------------------------------------------------------------+
      */
+    refda_agent_t *agent  = ctx->prodctx->parent->agent;
+    cace_ari_t     result = CACE_ARI_INIT_UNDEFINED;
+    atomic_ullong  val    = atomic_load(&agent->instr.num_rptset_sent);
+    cace_ari_set_uvast(&result, val);
+    refda_edd_prod_ctx_set_result_move(ctx, &result);
     /*
      * +-------------------------------------------------------------------------+
      * |STOP CUSTOM FUNCTION refda_adm_ietf_dtnma_agent_edd_num_msg_tx BODY
      * +-------------------------------------------------------------------------+
      */
 }
 
+// FUTURE:
```

Comment:
We can adjust the ADM and/or counters to capture what is available and remove what is not.
It is actually perfectly valid for an agent to leave the result as the undefined value as a marker of "I do not know this value".

Reply:
Appreciate your insight on this. If its OK with you I would prefer to leave the stub as-is for now and flesh it out under a different PR.